src/Kuzzle.js

```diff
@@ -48,19 +50,20 @@ function Kuzzle (host, options, cb) {
     connectCB: {
       value: cb
     },
-    eventListeners: {
-      value: {
-        connected: {lastEmitted: null, listeners: []},
-        error: {lastEmitted: null, listeners: []},
-        disconnected: {lastEmitted: null, listeners: []},
-        reconnected: {lastEmitted: null, listeners: []},
-        jwtTokenExpired: {lastEmitted: null, listeners: []},
-        loginAttempt: {lastEmitted: null, listeners: []},
-        offlineQueuePush: {listeners: []},
-        offlineQueuePop: {listeners: []},
-        queryError: {listeners: []},
-        discarded: {listeners: []}
-      }
+    eventActions: {
+      value: [
+        'connected',
+        'kerror',
```

Comment:
`error` events have a special behaviour in Node, and throw an error if it is triggered with no listener (see https://nodejs.org/api/events.html#events_error_events )
So we need another event name for Kuzzle errors to avoid bad behaviour.

Looking for a better name, any suggestion is welcome ;)